HW10/homework10.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+//HOMEWORK10
+
+// Задача на обробники подій, роботу зі сховищами та атрибутами/вмістом
+
+document.addEventListener('DOMContentLoaded', () => {
+
+  const button = document.querySelector('.btn');
+  const body = document.querySelector('body');
+  const text = document.querySelector('.text');
+
+  let isTurnedOn = (localStorage.getItem('isTurnedOn') !== 'false') ? true : false;
```

Comment:
А в якій умові виходить так що в локал сторадж сетится значення false у вигляді рядка?

Я б радив уникати таких варіантів

І Краще вже так:
```suggestion
  let isTurnedOn = !!localStorage.getItem('isTurnedOn');
```